README.md

```diff
@@ -1,24 +1,153 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- [📗 Table of Contents](#-table-of-contents)
+- [Budget App ](#budget-app-)
+  - [video demo 🎥](#video-demo-)
+  - [Render Deploy 🚀](#render-deploy-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [Acknowledgments](#acknowledgments)
+  - [📝 License ](#-license-)
 
-Things you may want to cover:
+* Configuration
+<!-- PROJECT DESCRIPTION -->
 
-* Ruby version
+# Budget App <a name="about-project"></a>
 
-* System dependencies
+> This mobile web app allows users to create categories and add transactions to each category. The app will show the total amount spent on each category. The app is built with Ruby on Rails. The app is deployed on Render.
+## video demo 🎥
 
-* Configuration
+[Video demo here](https://www.loom.com/share/72807faae3bd40759f6b9eaa28efb084)
+
+## Render Deploy 🚀
+
+[Live demo here](https://smart-spend.onrender.com/)
+
```

Comment:
- [ ] The video should contain a demo of the project, highlighting all the project features. I think somehow your video missed to record screen. Kindly do the change. :+1: